packages/ariakit/src/tooltip/__examples__/tooltip-fallback-placement/test-chrome.ts

```diff
@@ -0,0 +1,14 @@
+import { Page, devices, expect, test } from "@playwright/test";
+
+const getCanvas = (page: Page) =>
+  page.locator(`role=button[name='Tooltip'] >> xpath=..`);
+
+test("tooltip fallback placement", async ({ page }) => {
+  await page.goto("/examples/tooltip-fallback-placement");
```

Comment:
By default, playwright generates different screenshot files depending on the OS. This doesn't work well here. So let's remove the suffix (need to manually remove the old file).

```suggestion
test("tooltip fallback placement", async ({ page }, testInfo) => {
  testInfo.snapshotSuffix = "";
  await page.goto("/examples/tooltip-fallback-placement");
```